angular-pagedown.js

```diff
@@ -21,6 +21,7 @@ angular.module("ui.pagedown", [])
         restrict: "E",
         scope: {
             content: "=",
+            placeholder: "=",
```

Comment:
Do we need bi-directional binding on the placeholder attribute? 

If we do, the `link` function should watch its changes and update the DOM. Otherwise we should use "@" here for interpolated value.